lib/mapObjects/CGHeroInstance.cpp

```diff
@@ -320,7 +320,7 @@ void CGHeroInstance::initHero(CRandomGenerator & rand)
 	if (patrol.patrolling)
 		patrol.initialPos = visitablePos();
 
-	if(exp == 0xffffffff)
+	if(exp == std::numeric_limits<TExpType>::max())
```

Comment:
@rilian-la-te you changed value of constant `UNINITIALIZED_EXPERIENCE` (or what was its name) from `0xffffffff` to `std::numeric_limits<TExpType>::max()`

However, TExpType is uint64_t, meaning that this was change of behavior.

Good change, since otherwise we could end up with weird behavior if hero experience ends up exactly on `0xffffffff`. However code apparently had few other places where `0xffffffff` constant was used directly.

Reply:
wouldn't it make more sense to use the named constant instead of numeric_limits directly?

Reply:
There is no constant name AFAIK, number is used directly.

Maybe there is a reason to add such a constant, and name it exactly what you suggests?

Reply:
@kambala-decapitator +1

Reply:
Yeah, makes sense, will do.